OpenNMT/onmt/Models.py

```diff
@@ -29,10 +29,9 @@ def __init__(self, opt, dicts):
             self.word_lut.weight.copy_(pretrained)
 
     def forward(self, input, hidden=None):
-        emb = self.word_lut(input)
-
+        batch_size = input.size(0) # batch first for multi-gpu compatibility
+        emb = self.word_lut(input).transpose(0, 1)
```

Comment:
Backing up to before this PR was merged, looks like it ran about 300 tokens/s faster than after the merge (with all the transposes). I'll work on making everything batch first, and see if that makes up the difference.